### PR TITLE
dev-util/staruml-bin: new ebuild

### PR DESCRIPTION
**Pull request by Hummer12007**
A proprietary cross-platform UML editor in Node.js: http://staruml.io/ .

initial version: 2.7.0

new license: StarUML-EULA

### CODE REVIEW

File: dev-util/staruml/staruml-2.7.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+SRC_URI="
+	amd64? ( http://staruml.io/download/release/v${PV}/StarUML-v${PV}-64-bit.deb )
+	x86? ( http://staruml.io/download/release/v${PV}/StarUML-v${PV}-32-bit.deb )
+"
+
+LICENSE="StarUML-EULA no-source-code"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+RESTRICT="splitdebug"
```
Comment (SoapGentoo) [inline dev-util/staruml/staruml-2.7.0.ebuild]: is this a binary package? If so, it needs to be called `staruml-bin`

Comment (Hummer12007) [inline dev-util/staruml/staruml-2.7.0.ebuild]: @SoapGentoo done.

**Comment by monsieurp** [comment]
There has been some concerns over the EULA license added by this PR, particularly @leio who does not think it is appropriate for Gentoo. He will take this PR from there. Thank you.

**Comment by leio** [comment]
No, I am not taking this anywhere. https://devmanual.gentoo.org/general-concepts/licenses/ missing profiles/license_group entry.

**Comment by Hummer12007** [comment]
@leio ok now?

**Comment by leio** [comment]
It's case insensitive alphabetically sorted in that listing; keep it that way :)

**Comment by Hummer12007** [comment]
@leio

**Comment by leio** [comment]
Looks better now wrt license. I'm not part of proxy-maintainers to merge this personally, sorry. I also have not reviewing it in any other aspects, just the license bit something I noticed from main tree git pushes.

**Pull request closed without merging**